### PR TITLE
Revise e2e port-forward function

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Add e2e port-forward utility function.

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/util/portforward/portforward.go [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+	forwardedPorts, err = fw.GetPorts()
+	if err != nil {
+		cancel()
+		return nil, nil, err
+	}
+
+	return forwardedPorts, cancel, nil
+}
+
+// Forward would port-forward forward target resources
```
Comment (yeya24) [inline test/e2e/util/portforward/portforward.go:111]: `port-forward forward`?

Comment (Yisaer) [reply test/e2e/util/portforward/portforward.go]: just a typo fixed now.

**Pull request merged**